tools/bag_processing/scripts/rosbag_rewrite_types.py

```diff
@@ -153,12 +153,8 @@ def fix_message_definitions(inbag, fix_topic_patterns, verbose=False):
             logging.info("  [none]")
 
 
-def rename_types(inbag, outbag, rename_lookup, verbose=False):
-    """
-    Modifies inbag metadata (in memory, not on disk). Changes will take
```

Comment:
Did you mean to delete the comment?

Reply:
Yeah. That comment is the function description. Previously I thought the comment was important to help you understand what step the function is doing vs. what step remains for you to do later. Now the function does both steps, so it's less relevant.